civet/compilers/coffeescript.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function
+import json
+import os
+
+from django.conf import settings
+
+from civet.compilers.base_compiler import Compiler
+
+
+class CoffeescriptCompiler(Compiler):
+    def __init__(self, precompiled_assets_dir, kill_on_error):
+        super(CoffeescriptCompiler, self).__init__(precompiled_assets_dir,
+                                                   kill_on_error)
+        self.args = getattr(
+            settings, 'CIVET_COFFEE_SCRIPT_ARGUMENTS', ('--compile', '--map'))
+
+    @property
+    def name(self):
+        return "CoffeeScript"
+
+    @property
+    def executable_name(self):
+        return 'coffee'
+
+    @property
+    def executable_setting(self):
+        return 'CIVET_COFFEE_BIN'
```

Comment:
Can these three properties just be class variables, since they're constants? The base class can continue to define them as properties, since we'd want it to raise the `NotImplementedError`, but the subclasses can just redefine them as class variables.